### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by mariamicioni**

### CODE REVIEW

File: app/controllers/bears_controller.rb [hunk 1]
```diff
@@ -11,40 +10,37 @@ def show
...
-  if @bear.save
-    redirect_to bear_path(@bear)
-  else
-    render 'new'
+  def create
+    @bear = Bear.new(bear_params)
+    if @bear.save
+      redirect_to bear_path(@bear)
+    else
+      render 'new'
```
Comment (natebans) [inline app/controllers/bears_controller.rb:19]: maybe add render :new_, status: :unprocessable_entity_ so the user will also get an error message

File: app/controllers/bears_controller.rb [hunk 2]
```diff
@@ -11,40 +10,37 @@ def show
...
-  redirect_to bears_path
-end
-
-private
+  def update
+    @bear = Bear.find(params[:id])
+    if @bear.update(bear_params)
+      redirect_to bear_path(@bear)
+    else
+      render 'edit'
```
Comment (natebans) [inline app/controllers/bears_controller.rb:32]: same as above unprocessable entity comment

**Comment by natebans** [comment]
Hi Maria, looks good for the most part! before I merge, check my comments in the "Files changed" and let me know what you think 🙂

**Comment by natebans** [comment]
Looks great!

File: app/controllers/bears_controller.rb (hunk 2 above)
Comment (mariamicioni) [reply app/controllers/bears_controller.rb]: added

**Pull request merged**